staging/src/k8s.io/client-go/tools/auth/exec/exec_test.go

```diff
@@ -0,0 +1,205 @@
+/*
```

Comment:
How would we feel about a test that validates that the `rest.Config` returned from `LoadExecCredential` can actually be used to make a REST call? I worry that we will change something in `rest.ExecClusterToConfig` and forget to validate that `LoadExecCredential` still returns a REST config that is valid for HTTP calls. Perhaps this is being too wary...